src/ConnectApp.ts

```diff
@@ -0,0 +1,159 @@
+import { AuthError, AuthErrorCode } from './AuthError';
```

Comment:
I have renamed `Addon` to `ConnectApp` because it's what we call such things now.